src/app/space/create/codebases/codebases-item/codebases-item.component.ts

```diff
@@ -14,11 +14,11 @@ import { NotificationType as NotificationTypes } from 'patternfly-ng';
   styleUrls: ['./codebases-item.component.less']
 })
 export class CodebasesItemComponent implements OnDestroy, OnInit {
+  @Input() cheRunning: boolean;
```

Comment:
Where are we using cheRunning in this class? Please replace the broadcast subscription (cheStateChange') in ngOnInit. The cheRunning variable should be enough?